config/config_docker.json

```diff
@@ -20,7 +20,9 @@
         "InviteSalt": "gxHVDcKUyP2y1eiyW8S8na1UYQAfq6J6",
         "PublicLinkSalt": "TO3pTyXIZzwHiwyZgGql7lM7DG3zeId4",
         "ResetSalt": "IPxFzSfnDFsNsRafZxz8NaYqFKhf9y2t",
-        "AnalyticsUrl": ""
+        "AnalyticsUrl": "",
+        "UseLocalStorage": true,
+        "StorageDirectory": "/mattermost/"
```

Comment:
Docker installs us in /go/src/github.com/mattermost/platform and yeah /mattermost/data might be better as it gives a better indication as to what's being stored